doc/model/model.rst

```diff
@@ -21,36 +24,230 @@ Model
+
+2nd order terms on the viscous RHS
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The viscous fluxes $\mathbf{F}^{V}$ are proportional to gradients of the fluid state variables,
+introducing 2nd order terms on the RHS of the conservation equations. These 2nd order terms with their
+relevant rhs component are summarized below.
+
+Momentum equation
+"""""""""""""""""
+The 2nd order terms in the viscous RHS for the moementum equation are:
 
 .. math::
-    \frac{\partial \rho}{\partial t} + \frac{\partial }{\partial x_j} \rho u_j &= S_\rho \\
-    \frac{\partial \rho u_i}{\partial t} + \frac{\partial}{\partial x_j}\left(\rho u_i u_j + p\delta_{ij} - \tau_{ij}\right) &= S_{\rho u_i} \\
-    \frac{\partial \rho E}{\partial t} + \frac{\partial}{\partial x_j}\left(\left\{\rho E + p\right\}u_j + q_j - u_i \tau_{ij}\right) &= S_{\rho E},
+   \partial_j \tau_{ij} = \left[\partial_j\left(\mu\partial_j{v}_i\right) +
+   \partial_j\left(\mu\partial_i{v}_j\right) + \partial_j\left(\mu_{B} -
+   \frac{2}{3}\mu\right)\partial_k{v}_k\delta_{ij}\right]
 
-where $p$ is the thermodynamic pressure, $\tau_{ij}$ is the
-viscous stress tensor, and $q_i$ is the heat flux in the $i$th
-direction. $S_\rho$, $S_{\rho u_i}$, and $S_{\rho E}$ are are mass, momentum, and energy density source terms.  These equations can be written in the compact form
+
+Energy equation
+"""""""""""""""
+The 2nd order terms in the energy equation RHS have convective, conductive, and
+diffusive terms as follows:
+
+- Convective part
 
 .. math::
+   \partial_j \tau_{jk} {v}_k = \left[\partial_j\left(\mu\partial_k{v}_j{v}_k\right) +
+   \partial_j\left(\mu\partial_j{v}^2_k\right) + \partial_j\left(\mu_{B} -
+   \frac{2}{3}\mu\right)\partial_m{v}_m\delta_{jk}{v}_k\right]
+   
+
+- Conductive part
 
-    \frac{\partial Q}{\partial t} + \frac{\partial \vec{F}_j}{\partial x_j} = S,
+The conductive heat part of the RHS is:
+
+.. math::
+   \partial_j{(q_{c})_j} = \partial_j\kappa\partial_j{T},
 
-where $Q = [\rho\,\rho \vec{u}\,\rho E]^T$ is the vector of conserved
-variables, $\vec{F} = \vec{F}^I - \vec{F}^V$ is the flux vector account
-for both visicd and inviscid terms, and $S$ is the source term vector.
+where $T$ is the fluid temperature.
 
+- Diffusive part
 
+The diffusive heat part of the RHS is:
 
-.. toctree::
+.. math::
+   \partial_j{(q_{d})_j} = \partial_j\left(\rho{h}_{\alpha}{d}_{(\alpha)}\partial_j{Y}_{\alpha}\right)
+   
+with fluid density $\rho$, species diffusivity ${d}_{(\alpha)}$, and species mass fractions
+${Y}_{\alpha}$. 
+
+Species equation
+""""""""""""""""
+The species diffusive transport RHS is:
+
+.. math::
+   \partial_j{(J_{\alpha})_j} = \partial_j\left(\rho{d}_{(\alpha)}\partial_j{Y}_{\alpha}\right),
 
-   viscous
-   heat-flux
-   transport
-   power-law
-   eos
-   ideal-gas
-   non-dimen
+with fluid density $\rho$, species diffusivity ${d}_{(\alpha)}$, and species mass fractions
+${Y}_{\alpha}$. 
```

Comment:
I think all of this is more or less just restating what you've already defined above in sections 1.1--1.4 (except it's written in summation notation). Maybe you can write out the component-wise form next to the vector-calculus form (side-by-side) and use the newly created space to add a quick summary of the DG strategy?

Reply:
This entire section on the non-linear terms can be omitted, actually.  I meant for this to be an aid for our discussion about either using the diffusion operator in several different RHS's or potentially extending for tensor diffusivity and doing all the viscous terms with the diffusion operator.

Reply:
I left the section in for now (I just don't want to lose the text), and added a placeholder for the discretization doc. (e0c424f)